contracts/DependencyAudit.cdc

```diff
@@ -87,6 +159,17 @@ access(all) contract DependencyAudit {
             emit PanicOnUnstagedDependenciesChanged(shouldPanic: shouldPanic)
         }
 
+        // setStartEndBlock sets the start and end block heights for the `shouldPanicRandomly` function
+        access(all) fun setStartEndBlock(start: UInt64, end: UInt64) {
```

Comment:
Do we need any sort of input validation here? For instance, `start > end` or `getCurrentBlock().height <= end` etc.

Reply:
added